virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
```

Comment:
it looks you need a function like verify_support commands and call it before really execute the command. verify_supported should raise CommandNotSupport error if command not support by current guest agent.

Thanks,
Xu